grype/matcher/common/distro_matchers_test.go

```diff
@@ -53,6 +53,33 @@ func (pr *mockDistroProvider) stub() {
 			},
 		},
 	}
+	pr.data["sles:12.5"] = map[string][]vulnerability.Vulnerability{
+		// direct...
+		"sles": {
```

Comment:
nit: the key to this map should be a package name. What you've selected here will work given the package name used to validate is called `sles`, however, it's a little surprising the see the distro name in place of the package name.

Reply:
I've made that change.